src/models/game.js

```diff
@@ -91,4 +92,13 @@ export class Game {
       return dateA - dateB;
     });
   }
+
+  updateMissingProperties(details) {
+    if (details.releaseDate !== "") this.releaseDate = details.releaseDate;
+    if (details.developers.length !== 0) this.developers = details.developers;
```

Comment:
you should copy by value here instead of assigning the reference, also in line 100